components/server/src/api/workspace-service-api.ts

```diff
@@ -92,4 +99,80 @@ export class WorkspaceServiceAPI implements ServiceImpl<typeof WorkspaceServiceI
         response.pagination.total = resultTotal;
         return response;
     }
+
+    async createAndStartWorkspace(req: CreateAndStartWorkspaceRequest): Promise<CreateAndStartWorkspaceResponse> {
+        // We rely on FGA to do the permission checking
```

Comment:
Permission checking is inside WorkspaceService

Reply:
:heavy_check_mark: